### DISCUSSION TITLE
ERROR: Unable to query OneDrive to initialize application

### DISCUSSION THREAD

**Discussion by ManfredBartz** [abraunegg/onedrive#2804] [General]
I just installed the latest onedrive on Debian 12.  However, onedrive --synchronize gives me an error.
The Onedrive account is setup with 2FA - could that be an issue?
The tail-end of debug_output.log:
```
  OneDrive returned a 'HTTP 403 - Forbidden' - gracefully handling error
  [DEBUG] oneDriveRootDetails = onedrive.getDefaultRoot() generated a OneDriveException
  ERROR: Unable to query OneDrive to initialize application
  [DEBUG] OneDrive Account Details:      null
  [DEBUG] OneDrive Account Root Details: null
```

**Comment by abraunegg** [comment]
@ManfredBartz 

Please start from 'scratch' with these 2 commands only:

* `onedrive --logout`
* `onedrive --reauth`

If you cannot reauth, you cannot query your account and data - which means whatever account you are using has an issue. Please raise your account problem with your account provider.

**Comment by abraunegg** [comment]
@ManfredBartz 
FYI - Please **URGENTLY** change your account password. You left your authentication credentials in your debug log as this step was not followed:

![image](https://github.com/user-attachments/assets/d6d7f8b2-3432-416a-9cf1-fb0c09e40a85)
